check/README.md

```diff
@@ -0,0 +1,99 @@
+# Tool for work with string tables
+
+## Glossary
+
+- `string table` a file with translation
+- `reference table` en.txt string table.
+  English is the main game language, so any changes related to text should be in that file.
+  Also this table us used as fallback for other translation.
+  This file is mostly maintained by people who change game code,
+  so the list of entries in this file should be the most actual.
+- `entry` one piece of translation, it consists of
+  - `key` a sting that represents entry in the codebase (C++, Python).
+    This string must be unique.
+  - `value` actual translation that will be shown to the user.
+
+
+### Key
+Only English chars in uppercase and dash are allowed.
```

Comment:
English chars in uppercase and dash -> uppercase Latin alphabet (**A-Z**) and underscore (**_**)